### ISSUE TITLE
fill distance to polygons - TBD

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#2365]
Implement a new strategy to distance to polygons that consider the distance to the closest part of the polygon, instead of to its centroid.

**Comment by avancinirodrigo** [comment]
@pedro-andrade-inpe
could it be a parameter on distance operation in fill, e.g. `centroid = true`?

**Comment by pedro-andrade-inpe** [comment]
Yes, but we need to check if this could not have a bad interpretation in other strategies. Please check that.
